samples/asgardeo-react-app/src/config.json

```
@@ -3,5 +3,5 @@
     "baseUrl": "",
     "signInRedirectURL": "https://localhost:3000",
     "signOutRedirectURL": "https://localhost:3000",
-    "scope": ["profile"]
+    "scope": ["internal_login"]
```

Let's keep both scope values.

```suggestion
    "scope": ["profile", "internal_login"]
```